RedHat-UX/red-hat-design-system
# docs(logo-wall): use container queries, fix overflow

## What I did

1. Logo wall's lightdom CSS file uses only `@container` queries now.
2. All `@container` queries use `min-width`.
3. Fixed the overflow issue referenced in #1782.
4. Moves the `container-name` inside Logo wall's lightdom CSS file (helps when people copy and paste).
5. Wraps content in `.logo-wall-container` so that we can reference a `container` for the container queries.

## Testing Instructions

1. View the DP > Logo Wall > [Examples](https://deploy-preview-1800--red-hat-design-system.netlify.app/patterns/logo-wall/examples/).
2. See if the border intersects any logos at various screen sizes.
3. Verify if the original design has been maintained. 
4. See changes in the lightdom CSS file and raise any issues, if any.

## Notes to Reviewers

Some `@container` queries use `min-width` values that are not token values. I don't think there's a way around this because the smallest token value is 575px and the layout necessitates multiple columns at smaller than 575px container sizes.

Closes: #1782
